testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -481,14 +506,15 @@ protected void checkInformationSchemaTablesForPointedQueryForMaterializedView(St
     }
 
     // TODO inline when all implementations fixed
-    protected void checkShowColumnsForMaterializedView(String schemaName, String viewName)
+    protected void checkShowColumnsForMaterializedView(String viewName)
     {
         assertThat(query("SHOW COLUMNS FROM " + viewName))
+                .projected(0)
```

Comment:
would matching column type be hard to do instead?

Reply:
Since column types can vary between connectors, trying to do that in BCT could mean overriding implementations in individual connectors

Reply:
Please add a comment about it